plugins/strace.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: strace
+spec:
+  version: "v0.0.1"
+  homepage: https://github.com/michaelwasher/kstrace
```

Comment:
I am noticing you did not import client-go auth plugins?
https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/#:~:text=in%20your%20plugins.-,Import%20authentication%20plugins%20(Go),-By%20default%2C%20plugins

Reply:
Hi @ahmetb,

Thank you for letting me know. Sorry I completely missed it. I'll fix that up.
Also, regarding the use of genericclioptions, is it an expectation to include the Kuebrentes default options(i.e. `-kubeconfig`, `--context`) or can this be excluded how I've done it here: https://github.com/MichaelWasher/kstrace/blob/master/cmd/cmd.go#L119-L122

Reply:
This import has been added, the release re-made and the SHA's updated.

Reply:
We don't have any enforcement of genericclioptions (it's a recommendation). Same with client-go/pkg/client/auth –but if you don't import this, you're severely limiting your user base.